e2e-tests/config/substrate/ci_nodes.json

```diff
@@ -7,8 +7,8 @@
   "nodes_config": {
     "nodes": {
       "validator-1": {
-        "host": "10.0.10.7",
-        "port": "30621",
+        "host": "ci-preview-validator-1-service.ci-preview.svc.cluster.local",
```

Comment:
FQDN only works from cluster, right? This config is to run tests from local. When we run tests in CI we use `--ci-run` and a separate config overrides IPs with FQDNs, see `*-ci.json`

Reply:
the other thing is that we're missing `ci-ci.json` config